SpatialGDK/Documentation/content/technical-overview/gdk-concepts.md

```diff
@@ -89,24 +89,24 @@ You create a Singleton Actor by tagging an Actor with the `SpatialType=Singleton
 
 For more information, see the documentation on [Singleton Actors]({{urlRoot}}/content/singleton-actors).
 
-### Actor Throttling
-We have introduced a concept of 'Actor Throttling' in order to optimize server performance when using the Unreal GDK, it can be accessed via the [Runtime Settings]({{urlRoot}}/content/unreal-editor-interface/runtime-settings). With the Unreal GDK it is possible to have multiple servers simulating on the same game world, this means that no one server could have a full view of the entire world at once. Additionally, servers in the Unreal GDK replicate actors only to a single connection to SpatialOS rather than to each individual client as in native Unreal. To accommodate this, in order to keep the state of the SpatialOS simulation accurate, all actors which are in a particular server's view are marked as relevant for replication on each frame. This can however cause performance issues if there are say, 3000 actors on a single server. 
+### Actor throttling
+We have introduced a concept of 'Actor throttling' in order to optimize server performance when using the Unreal GDK, it can be accessed via the [Runtime Settings]({{urlRoot}}/content/unreal-editor-interface/runtime-settings). With the Unreal GDK it is possible to have multiple servers computing on the same game world, this means that no one server could have a full view of the entire world at once. Additionally, servers in the Unreal GDK replicate actors only to a single connection to SpatialOS rather than to each individual client as in native Unreal. To accommodate this, in order to keep the state of the SpatialOS simulation accurate, all actors which are in a particular server's view are marked as relevant for replication on each frame. This can however cause performance issues if there are say, 3000 actors on a single server. 
```

Comment:
Same comment as before about chatty blogpost style text.
Shall we make a ticket for this in the fallout tasks because this is signficant?

Reply:
As above, non-tech-writered (and I've already created a ticket :) https://improbableio.atlassian.net/browse/DOC-1265